sql/codegen_alps.go

```diff
@@ -195,6 +194,109 @@ func submitALPS(w *PipeWriter, pr *extendedSelect, db *DB, cwd string) error {
 	return nil
 }
 
+func (nc *numericColumn) GenerateAlpsCode(metadata *Metadata) ([]string, error) {
```

Comment:
How is this function different from 
https://github.com/sql-machine-learning/sqlflow/blob/3867f64c17e852671acadc5d4ab6d68438aba5cc/sql/expression_resolver.go#L641-L644

Reply:
GenerateCode return (string, error)
GenerateAlpsCode return ([]string, error)

Reply:
for multi group,
category_column(deep) needs return category_column(deep0), category_column(deep_1) ...
so i introduce generateAlpsCode.

I thinks GenerateCode be a function of feature column is a bad idea. GenerateCode should be a part of physical-plan.